packages/data-models/constants.ts

```diff
@@ -30,3 +30,12 @@ export const APP_FIELDS = {
   SERVER_SYNC_LATEST: `${FIELD_PREFIX}._server_sync_latest`,
   APP_LANGUAGE: `${FIELD_PREFIX}._app_language`,
 };
+
+/**
+ * Some specific strings are currently hardcoded into the app
+ * TODO - not all strings included, should add to when required
+ */
+export const APP_STRINGS = {
+  NOTIFICATION_DEFAULT_TITLE: "Notification",
+  NOTIFICATION_DEFAULT_TEXT: "You have a new message from PLH",
+};
```

Comment:
@EsmeeIDEMS Can you define these with Anna it is important that even this text gets approval at least form the content authors

Reply:
I've contacted Anna about this, but it might take a while before I hear back from her. I'm happy for this to be merged with the current defaults, I can change them afterwards.